core/arch/x86/mangle.c

```diff
@@ -3858,18 +3858,23 @@ sandbox_top_of_bb(dcontext_t *dcontext, instrlist_t *ilist,
                                  OPND_CREATE_INTPTR(end_pc - (start_pc + 1))));
         PRE(ilist, instr,
             INSTR_CREATE_jcc(dcontext, OP_jge, opnd_create_instr(forward)));
+        /*
```

Comment:
style, comment start at the first line.
https://github.com/DynamoRIO/dynamorio/wiki/Code-Style#commenting-conventions